doc/whats_new.rst

```diff
@@ -17,7 +17,7 @@ Changelog
 - Store all connectivity information under :attr:`~hnn_core.Network.connectivity` before building
   the network, by `Nick Tolley`_ in :gh:`276`
 
-- Add new function :func:`~hnn_core.viz.plot_cell_morphology` to visualize cell morphology, 
```

Comment:
Hmm, this is curious. I don't recall touching any other items in whats_new.rst besides my own entry. Is it possible that my editor did something automatically?

Reply:
it just removed the trailing whitespace which is fine